cypress/integration/smoke/tx_list.spec.js

```diff
@@ -0,0 +1,133 @@
+const STATIC_TEST_SAFE = 'rin:0x1230B3d59858296A31053C1b8562Ecf89A2f888b'
+
+describe('Transaction List', () => {
+  it('should display the transaction list', () => {
+    cy.visit(`/${STATIC_TEST_SAFE}/transactions/history`)
+
+    // Accept cookies
+    cy.findByText('Accept selection').click()
+
+    // Opening/closing of accordion
+    cy.findByText('changeThreshold').click().click()
+    cy.get('.tx-data').should('not.exist')
+
+    const TX_975_TITLE = 'Gnosis Safe: MultiSendCallOnly'
+
+    // Infinite scroll
+    cy.findByText(TX_975_TITLE).should('not.exist')
+    cy.get('#infinite-scroll-container').scrollTo('bottom')
+    cy.wait(3000)
+    cy.findByText(TX_975_TITLE).should('exist')
```

Comment:
That's just implementation details, we shouldn't test that.

Reply:
The transaction being tested is not on the first page.